gui/wxpython/modules/import_export.py

```diff
@@ -60,7 +60,8 @@ def __init__(self, parent, giface, itype,
         self.options = dict()   # list of options
         self.options_par = dict()
 
-        self.showImportSuccessfulInfo = Signal('GdalImportDialog.showImportSuccessfulInfo')
+        self.switchToDataTab = Signal('ImportDialog.switchToDataTab')
+        self.showImportSuccessfulInfo = Signal('ImportDialog.showImportSuccessfulInfo')
```

Comment:
What about one Signal for both? They seem to be pretty much the same. The point of the signals is that the class which emits them doesn't know anything about the actual usage. Here it seems really just as calling specific functions, but doing it through the signals.

Reply:
Makes sense, but the behaviour should be consistent. We need to decide whether we want to show the third infobar and switch to Data tab when calling specific functions from File menu. Or do not want to change lgmr.py at all and focus only on calling using buttons in infobar and Management icons. 

This PR deals with the third Infobar, so probably the best way would be to not solve File tab here at all. So, the switch to Data tab and show of infobar would be conditioned by calling the functions either from infobar buttons, or from Management icons for import (https://github.com/OSGeo/grass/pull/1204 (still open)).

Reply:
This waits for #1205.